Makefile.docker

```diff
@@ -1,3 +1,11 @@
+# Makefile.docker contains the shared tasks for building, tagging and pusshing Docker images. 
+# This file is included into the Makefile files which contain the Dockerfile files (E.g. 
+# kafka-base, kafka-statefulsets etc.).
+#
+# The DOCKER_ORG (default is name of the current user) and TAG (default is latest) variables 
```

Comment:
Here you are mentioning TAG. Are you referring to VERSION ? Can we use a consistent naming ?

Reply:
Actually, the tag of the docker image is controller by the TAG variable not VERSION. And in CI the TAG variable should correspond to the Git tag of the build version. This will make sure that when we tag the release in gitHub, the CI automatically pushes the docker images with the right versions. The VERSION is based on Git commit hash and is injected into the Docker image.

I also use a VERSION variable for the release, which might be a bit confusing. But the release is currently done manually, so it has to be anyway configured by whoever runs the release.

Reply:
Ok, I renamed most of the parameters to make it a bit more clear what are they for. Please have a look if it is now a bit more undertsandable.